dags/catalyst.py

```diff
@@ -0,0 +1,46 @@
+"""
+DAG to schedule generation of performance reports for recently completed nimbus experiments.
+
+See the [jetstream repository](https://github.com/mozilla/catalyst).
+
```

Comment:
It may be helpful to add a section to these docs for Airflow Triage called `*Triage Notes*`. Airflow Triage is a rotating responsibility for DEs to ensure that DAG failures are noticed and documented in a bug. The triage person may also attempt to diagnose and resolve issues when possible.

If you're not sure of any patterns or specific advice now, you can always update this later if any patterns emerge.